### ISSUE TITLE
LBFGS learning rate

### ISSUE THREAD

**Issue by ahmadajal** [rakutentech/stAdv#11]
Hi,

I was trying to reproduce your results in pytorch and I am struggling to get proper results with LBFGS. The issue is that it is not really clear for me how you set the learning rate for LBFGS and what was the learning rate which was used to produce the results in the paper. It is neither mentioned in the paper nor I could figure this out from the code. Could you please help me with that?
